collectors/CMPCollector.js

```diff
@@ -6,20 +6,32 @@ const BaseCollector = require('./BaseCollector');
 
 // @ts-ignore
 const baseContentScript = fs.readFileSync(
-    path.join(__dirname, "../node_modules/@duckduckgo/autoconsent/dist/autoconsent.standalone.js"),
+    path.join(__dirname, "../node_modules/@duckduckgo/autoconsent/dist/autoconsent.playwright.js"),
```

Comment:
That bundle is the same one used by our playwright regression tests, which is it's original and main purpose. It suits the crawler because it's puppeteer based, which is almost the same thing. 

We can't use that bundle in browsers because browsers need to map callbacks to platform specific apis like webkits messageHandlers